arch/arc/core/mpu/arc_core_mpu.c

```diff
@@ -9,7 +9,10 @@
 #include <kernel.h>
 #include <soc.h>
 #include <arch/arc/v2/mpu/arc_core_mpu.h>
-#include <logging/sys_log.h>
+
+#define LOG_LEVEL CONFIG_ARCH_LOG_LEVEL
```

Comment:
In my opinion we should have separate log level configuration for MPU (as it is separate subsystem).